en/core-api/e-wallet.md

```diff
@@ -1,76 +1,75 @@
+| Production  | POST   | https://api.midtrans.com/v2/charge         |
 
-*This is an example in Curl, please implement according to your backend language, you can switch to other language on the "tab" above. (you can also check our [available language libraries](/en/technical-reference/library-plugin.md))*
+#### Headers
 
-#### Request Details
-Type | Value
---- | ---
-HTTP Method | `POST`
-API endpoint (Sandbox) | `https://api.sandbox.midtrans.com/v2/charge`
-API endpoint (Production) | `https://api.midtrans.com/v2/charge`
+| Header Name   | Description                                            | Required | Values                |
+| ------------- | ------------------------------------------------------ | -------- | --------------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json      |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json      |
+| Authorization | The authentication method used to access the resource. | Required | Basic **AUTH_STRING** |
 
-#### HTTP Headers
-```
-Accept: application/json
-Content-Type: application/json
-Authorization: Basic AUTH_STRING
-```
+**AUTH_STRING**: Base64(`ServerKey + :`)<br>Midtrans API validates HTTP request by using Basic Authentication method. The username is your *Server Key* while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by a colon symbol (**:**). For more details, refer to [ API Authorization and Headers](https://docs.midtrans.com/en/technical-reference/api-header).
+
+?> ***Note***: *Server Key* is required to authenticate the request. For more details, refer to [HTTPS Header](https://api-docs.midtrans.com/#http-s-header).<br>
 
-**AUTH_STRING**: Base64(`ServerKey + :`)
+#### POST Body
 
-?> Core API validates HTTP request by using Basic Authentication method. The username is your Server Key while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by **:** (colon symbol).
+Element | Description |Type|Required
+----------- | ----------- | ----------- | ----------- 
+transaction_details | Details of the transaction such as order_id and gross_amount |-|Required
+order_id | Transaction order ID, defined from your side |String|Required
+gross_amount | Total amount of transaction, defined from your side |String|Required
+payment_type | Type of payment |String|Required
 
-#### Full HTTP Request
+#### Sample Request 
+
+The sample request for *charge API* Charge is as given below. APIs are implemented in some of the commonly used languages. You may implement according to your backend language. For more details, refer to available [Language Libraries](/en/technical-reference/library-plugin.md#language-library).
+<!-- tabs:start -->
+
+#### CURL
```

Comment:
The formatting is broken, please fix it, Have you check viewing from browser before submitting this pull request?

<img width="772" alt="Screen Shot 2020-11-03 at 16 35 05" src="https://user-images.githubusercontent.com/13027142/97969407-98560f00-1df2-11eb-9a65-e93470f100ae.png">


Also why taking out the Endpoint, Headers, & POST Body outside of the API-Request/CURL tabs? it is only relevant for API-Request/CURL, it is not relevant for other like PHP, NodeJS, etc. So it should stay within that tabs.